doc/api/async_hooks.md

```diff
@@ -529,15 +528,15 @@ class DBQuery extends AsyncResource {
 
 #### `asyncResource.emitBefore()`
 
-* Returns {undefined}
+* Returns: {undefined}
```

Comment:
Usually we just omit the return type if it's `undefined`, though I'm interested to hear another opinion.

Reply:
https://github.com/nodejs/build/pull/881

Reply:
I think given that our docs aren't 100% complete in terms of listing return types, being explicit here is probably helpful for people.